lib/slack/web/config.rb

```diff
@@ -21,15 +21,21 @@ module Config
       def reset
         self.endpoint = 'https://slack.com/api/'
         self.user_agent = "Slack Ruby Client/#{Slack::VERSION}"
-        self.ca_path = `openssl version -a | grep OPENSSLDIR | awk '{print $2}'|sed -e 's/\"//g'`
-        self.ca_file = "#{ca_path}/ca-certificates.crt"
+        self.ca_path = openssl_ca_path if ca_path.nil?
```

Comment:
Can't this be just using `OpenSSL::X509::DEFAULT_CERT_DIR` and `OpenSSL::X509::DEFAULT_CERT_FILE` ?

Reply:
This (openssl version -a) does not work on MacOS Sierra (10.12) also.

My `OPENSSLDIR` is `/System/Library/OpenSSL`. 

But `/System/Library/OpenSSL/ca-certificates.crt` does not exist.


When using system ruby then  `OpenSSL::X509::DEFAULT_CERT_FILE` points to `/System/Library/OpenSSL/cert.pem` which also does not exist.

But at least it works with own compiled ruby.

Also it is reasonable to use `SSL_CERT_DIR` and `SSL_CERT_FILE` env variables before the openssl default. (like https://github.com/mikz/httpclient/blob/ba21d0d44c9f5f4602c7fc151f7c24f827480361/lib/httpclient/ssl_config.rb#L418-L419) 

As those are used by OpenSSL to set custom cert dir/path.

Reply:
I'll change the pr to use OpenSSL::X509::DEFAULT_CERT_DIR and OpenSSL::X509::DEFAULT_CERT_FILE, since that works on nix, win and osx.